src/main/java/frc/robot/OI/OperatorInterface.java

```diff
@@ -69,48 +70,53 @@ public boolean CheckLowGear(boolean previous) {
   }
 
   public boolean getTurretAdjustLeft() {
-    return OperatorController.getButton(XboxController.Button.A);
+    return OperatorController.getDPad() == DPad.LEFT;
   }
 
   public boolean getTurretAdjustRight() {
-    return OperatorController.getButton(XboxController.Button.B);
+    return OperatorController.getDPad() == DPad.RIGHT;
   }
 
   // Operator
 
   public boolean getHarvestMode() {
-    return DriverController.getTrigger(XboxController.Side.LEFT);
+    return OperatorController.getButton(NykoController.Button.LEFT_TRIGGER);
   }
 
   public double getOperatorThrottle() {
-    return OperatorController.getJoystick(XboxController.Side.LEFT, XboxController.Axis.Y);
+    return OperatorController.getJoystick(NykoController.Side.LEFT, NykoController.Axis.Y);
   }
 
   public double getOperatorTurn() {
-    return OperatorController.getJoystick(XboxController.Side.RIGHT, XboxController.Axis.X);
-  }
-
-  public int getTurretManual() {
```

Comment:
Did we delete the method getTurretManual?

Reply:
Replaced by `getTurretAdjustLeft()` and   `getTurretAdjustRight()`